var/spack/repos/builtin/packages/minife/package.py

```diff
@@ -0,0 +1,93 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+
+import tarfile
+
+from spack import *
+
+
+class Minife(MakefilePackage):
+    """Proxy Application. MiniFE is an proxy application
+       for unstructured implicit finite element codes.
+    """
+
+    homepage = "https://mantevo.org/"
+    url      = "http://mantevo.org/downloads/releaseTarballs/miniapps/MiniFE/miniFE-2.0.1.tgz"
+
+    tags = ['proxy-app']
+
+    version('2.0.1', '3113d7c8fc01495d08552672b0dbd015')
+
+    variant('build', default='ref', description='Type of Parallelism',
+            values=('ref', 'openmp', 'qthreads', 'kokkos'))
+
+    depends_on('mpi')
+    depends_on('qthreads', when='build=qthreads')
+
+    @property
+    def type_of_build(self):
+        build = 'ref'
+
+        if 'build=openmp' in self.spec:
```

Comment:
I will make that change.  I am working on a similar package where the names use some uppercase.  Does spack want the variant values to be lowercase?